packages/create-astro/src/actions/dependencies.ts

```diff
@@ -27,16 +27,17 @@ export async function dependencies(
 		await spinner({
 			start: `Installing dependencies with ${ctx.packageManager}...`,
 			end: 'Dependencies installed',
+			onError: (e) => {
+				error('error', e);
+				error(
+					'error',
+					`Dependencies failed to install, please run ${color.bold(
+						ctx.packageManager + ' install'
+					)} to install them manually after setup.`
+				);
+			},
 			while: () => {
-				return install({ packageManager: ctx.packageManager, cwd: ctx.cwd }).catch((e) => {
-					error('error', e);
-					error(
-						'error',
-						`Dependencies failed to install, please run ${color.bold(
-							ctx.packageManager + ' install'
-						)}  to install them manually after setup.`
-					);
-				});
+				return install({ packageManager: ctx.packageManager, cwd: ctx.cwd }).catch((e) => { throw e });
```

Comment:
Do we need the `.catch((e) => { throw e })` part? I think it should already throw by default.

Reply:
yup, good catch, we don't and we can just return `install` function. Applied suggestion in the commit.

Tested manually few times, works as expected